src/store/modules/ProfileModule.ts

```diff
@@ -467,6 +467,7 @@ export default {
         },
 
         async updateModList({commit}, modList: ManifestV2[]) {
+            commit('tsMods/prewarmCacheMod', modList, { root: true });
```

Comment:
Nitpick: since updateModList is called in places where were already dealing with mods that are being used in the UI right away, this is not so much prewarming the cache as just updating it. Might consider renaming it at some point.